flang/include/flang/Optimizer/Transforms/Factory.h

```diff
@@ -137,14 +141,27 @@ void genCharacterCopy(mlir::Value src, mlir::Value srcLen, mlir::Value dst,
   builder.restoreInsertionPoint(insPt);
 }
 
+/// Get extents from fir.shape/fir.shape_shift op. Empty result if
+/// \p shapeVal is empty or is a fir.shift.
+inline std::vector<mlir::Value> getExtents(mlir::Value shapeVal) {
+  if (shapeVal)
+    if (auto *shapeOp = shapeVal.getDefiningOp()) {
+      if (auto shOp = mlir::dyn_cast<fir::ShapeOp>(shapeOp))
+        return shOp.getExtents();
+      if (auto shOp = mlir::dyn_cast<fir::ShapeShiftOp>(shapeOp))
```

Comment:
No. https://llvm.org/docs/CodingStandards.html#do-not-use-braced-initializer-lists-to-call-a-constructor

Since most of lowering and later stages are built upon and use MLIR and LLVM, the coding conventions of the ecosystem are used. Furthermore, it's distracting to have a contrarian to LLVM style. https://mlir.llvm.org/getting_started/DeveloperGuide/